pkgs/development/interpreters/clojure/babashka.nix

```diff
@@ -79,7 +79,7 @@ stdenv.mkDerivation rec {
     '';
```

Comment:
yeah it was only on the stable channel and I didn't notice as I allowBroken on my config because I'm investigating a different broken package right now.

Thanks for the review! I've added two more commits to fix the wrong atrribute.